docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -156,3 +205,8 @@ subpackage :code:`fairlearn.metrics`.
    .. [#3] Agarwal, Beygelzimer, Dudik, Langford, Wallach `"A Reductions
       Approach to Fair Classification"
       <https://arxiv.org/pdf/1803.02453.pdf>`_, ICML, 2018.
+	  
+   .. [#4] Selbst, Andrew D. and Boyd, Danah and Friedler, Sorelle and Venkatasubramanian, Suresh and Vertesi, Janet, "Fairness and Abstraction in     	  		Sociotechnical Systems" (August 23, 2018). 2019 ACM Conference on Fairness, Accountability, and Transparency (FAT*), 59-68, Available at `SSRN: 		<https://ssrn.com/abstract=3265913>`_, 
```

Comment:
Something is a little off about the line wrap here. Let's try to stay below 80 characters, unless it would interrupt a link/URL.

Same below, and you can continue the following line right below the `[#5]` the same way it's been done above with `[#3]`.

Reply:
Tried shortening it in my latest commit. Let me know if this works better.

Reply:
The second, third, etc. lines are still indented by 2 characters more than they'd need to be indented, but now I'm really nitpicking 😆 Sorry, I'll restrict myself to comments on the actual substance for now.

Reply:
Can you please confirm if the spacing looks more like what you had in mind @romanlutz ?

Reply:
Yes, looks great! Thank you @lgfunderburk!